src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -182,7 +181,7 @@ public final void testPathsDirectoryWithNestedFile() throws IOException {
         antTask.setProject(new Project());
 
         final FileResource fileResource = new FileResource(
-            antTask.getProject(), getPath(FLAWLESS_INPUT_DIR));
+            antTask.getProject(), getPath(FLAWLESS_INPUT));
```

Comment:
@rnveach Done. I found a way so that the test would throw an error and then made the necessary changes for it in the test.